docs/apispec-docs/src/main/scala/sttp/tapir/docs/apispec/schema/TSchemaToASchema.scala

```diff
@@ -21,16 +21,16 @@ private[schema] class TSchemaToASchema(nameToSchemaReference: NameToSchemaRefere
             required = p.required.map(_.encodedName),
             properties = fields.map { f =>
               f.schema match {
-                case TSchema(_, Some(name), _, _, _, _, _, _, _) => f.name.encodedName -> Left(nameToSchemaReference.map(name))
+                case TSchema(_, Some(name), _, _, _, _, _, _, _, _) => f.name.encodedName -> Left(nameToSchemaReference.map(name))
```

Comment:
I think that we're missing here the hidden check as well - for products, coproducts and open products we only want to include non-hidden child properties / schemas